massgov/mayflower-patternlab
# DP-8877  MF create front end code to support display of short description on org page that is separate from contact row

## Description
- Adds short description pattern below banner for Organization, Organization/Elected Official and Organization Board pages
- Updates order of elements to better match Drupal pages 

## Related Issue / Ticket

- [JIRA DP-8877](https://jira.state.ma.us/browse/DP-8877)
- [JIRA DP-8891](https://jira.state.ma.us/browse/DP-8891)

## Steps to Test
1. Visit the /?p=pages-organization, /?p=pages-organization-boards and /?p=pages-organization-elected-official pages to see the new description that is not part of the page-header pattern

## Screenshots
<img width="1151" alt="screen shot 2018-05-07 at 3 09 27 pm" src="https://user-images.githubusercontent.com/18662734/39722521-aaed0e38-5208-11e8-91b5-4ebc23567a0f.png">
